src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -64,6 +85,29 @@ public void arcadeDrive(double translate, double rotate) {
 
         setRawPower(left, right);
     }
+    
+    public void kinematicsDrive(double linearVelocity, double angularVelocity) {
+        var chassisSpeed = new ChassisSpeeds(linearVelocity, 0.0, angularVelocity); // 0.0 = no y directional
+        wheelSpeeds = kinematics.toWheelSpeeds(chassisSpeed);
+        
+        double leftPower = getLeftSpeedVoltage(wheelSpeeds)/leftMaxVoltage;
```

Comment:
Since we set the speeds directly, we don't need to divide by the maximum voltage to get a value in the -1,1 range.